src/nes/cartridge.nim

```diff
@@ -4,7 +4,7 @@ export types.Cartridge
 
 const iNESMagic = 0x1A53454E
 
-type iNESHeader = object {.packed.}
+type iNESHeader {.packed.} = object
```

Comment:
``/home/dian/Projects/nimes/src/nes/cartridge.nim(7, 26) Warning: type pragmas follow the type name; this form of writing pragmas is deprecated [Deprecated]``